### PR TITLE
Fix item use create measure template option not working with aoe spells

### PR DESCRIPTION
**Pull request by elwin6446**

### CODE REVIEW

File: module/applications/item/ability-use-dialog.mjs [hunk 1]
```diff
@@ -48,7 +49,7 @@ export default class AbilityUseDialog extends Dialog {
       consumeResource: resource.target && (!item.hasAttack || (resource.type !== "ammo")),
       consumeUses: uses.per && (uses.max > 0),
       canUse: recharges ? recharge.charged : sufficientUses,
-      createTemplate: game.user.can("TEMPLATE_CREATE") && item.hasAreaTarget,
+      createTemplate: game.user.can("TEMPLATE_CREATE") && (config?.createMeasuredTemplate ?? item.hasAreaTarget),
```
Comment (krbz999) [inline module/applications/item/ability-use-dialog.mjs:52]: Would this not cause an issue if passing `createMeasuredTemplate: true` while `item.hasAreaTarget` is `false`?
It seems like the purpose of the `config` object in this context is to explicitly only disable some properties in the `data` object, but here it is taking priority. Maybe adding `config.createMeasuredTemplate !== false` instead would work (or using `mergeObject`), like,
```js
createTemplate: game.user.can("TEMPLATE_CREATE") && item.hasAreaTarget && (config.createMeasuredTemplate !== false),
```

Comment (elwin6446) [reply module/applications/item/ability-use-dialog.mjs]: A change was pushed to make sure that it cannot be true if is not an AOE

**Comment by Fyorl** [comment]
What issue is this fixing? Is there an associated bug report in the tracker?

**Comment by elwin6446** [comment]
> What issue is this fixing? Is there an associated bug report in the tracker?

It fixes https://github.com/foundryvtt/dnd5e/issues/2289

**Comment by Fyorl** [comment]
Marking as blocked on #2387

**Comment by Fyorl** [comment]
This should now be resolved on the `2.4.x` branch and `AbilityUseDialog` will now be better-behaved about respecting options passed to it. Please feel free to re-open the original issue if not.

**Pull request closed without merging**